### ISSUE TITLE
Store component images in Airtable

### ISSUE THREAD

**Issue by dustin-jw** [sparkbox/accessible-components#139]
Having the component images in code will slow down the process for adding new components once the Accessibility Guild takes over the project. By moving them to Airtable, it should be easier to update them, or even use “Coming soon” images for components that aren’t ready without requiring code changes.

With the new hover states, we’ll need two new columns in Airtable:

- Default Image
- Hover Image

Then, we’ll need to update our getStaticProps function in [id].page.jsx to pass along those values to the details page and DetailsBanner component.

The image URLs on the home page will also need to be updated, but it’s okay for them to be hard-coded (rather than fetched from Airtable) for the scope of this card.

Definition of Done:

The component images are stored in Airtable, and the details page has been updated to show the right image based on the slug and the date we get from Airtable.

**Comment by dustin-jw** [comment]
Done!
